index.test.js

```diff
@@ -20,6 +22,22 @@ test('Query all records without metadata', async (t) => {
   t.ok(dataset.data, 'Returns all records without metadata');
 });
 
+test('Query all records without options', async (t) => {
+  const CMSClient = createClient('5fr6-cch3');
+
+  const dataset = await CMSClient.select().get();
+  t.ok(dataset.data, 'Returns all records with metadata');
+});
+
+test('Query all records with csv output', async (t) => {
+  const CMSClient = createClient('5fr6-cch3', {
+    output: 'csv',
+  });
+
+  const dataset = await CMSClient.select().get();
+  t.ok(dataset.data, 'Returns all records in csv format');
```

Comment:
According to [docs](https://node-tap.org/docs/api/asserts/), `t.ok` is just verifying that the value passed in is truthy. Is this actually testing if the function returns "all records in a csv format?" All good if you're just scaffolding some tests to add more complexity later. I'm cool with that.

Reply:
I am just scaffolding them right now. I am using `t.ok` to give me slight confidence between the changes I make. For the next PR, I am replacing `t.ok` with `t.equal` and also checking for the correct output type, as you mentioned. You are right; these tests are a bit misleading. I will fix it asap!